test/exportToNDJson.test.js

```diff
@@ -0,0 +1,38 @@
+const build = require('../src/server/app');
+const { exportToNDJson } = '../src/util/exportToNDJson';
+const { createTestResource, cleanUpDb } = './populateTestData';
+const testPatient = require('./fixtures/testPatient.json');
+const app = build();
+const fs = require('fs');
+const mockRequestWithType = {
+  query: {
+    _type: 'Patient'
+  }
+};
+const mockRequestWithoutType = {
+  id: 'mockRequestWithoutType',
+  query: {}
+};
+const clientId = '123456';
+//= ./tmp/ + type.toString(); + resourceType + clientId + '.ndjson';
+const expectedFileName = './tmp/Patient/patient123456.ndjson';
```

Comment:
```suggestion
const expectedFileName = './tmp/Patient/patient_123456.ndjson';
```
...if you add in my suggested `_` above